client/src/app/router.coffee

```diff
@@ -0,0 +1,24 @@
+Marionette = require('backbone.marionette')
+SiteView = require('./views/site')
+ListSitesView = require('./views/list-sites')
+FormView = require('./views/form')
+HomeView = require('./views/home')
+
+module.exports = Marionette.AppRouter.extend(
+
+  routes:
+    '': 'index'
+    'create': 'create'
+    'list_sites': 'list_sites'
+
+  index: ->
+    new HomeView().render()
```

Comment:
@samccone So apparently App Regions are also deprecated.  The recommendation is to use a LayoutView, which I will do.